bootstrap/bootstrapper/commitlog/commit_log.go

```diff
@@ -0,0 +1,51 @@
+// Copyright (c) 2016 Uber Technologies, Inc.
+//
+// Permission is hereby granted, free of charge, to any person obtaining a copy
+// of this software and associated documentation files (the "Software"), to deal
+// in the Software without restriction, including without limitation the rights
+// to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+// copies of the Software, and to permit persons to whom the Software is
+// furnished to do so, subject to the following conditions:
+//
+// The above copyright notice and this permission notice shall be included in
+// all copies or substantial portions of the Software.
+//
+// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+// IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+// AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+// LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+// OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+// THE SOFTWARE.
+
+package commitlog
+
+import (
+	"github.com/m3db/m3db/bootstrap/bootstrapper"
+	"github.com/m3db/m3db/interfaces/m3db"
+)
+
+const (
+	// CommitLogBootstrapperName is the name of the commit log bootstrapper.
+	CommitLogBootstrapperName = "commitlog"
+)
+
+type commitLogBootstrapper struct {
+	m3db.Bootstrapper
+}
+
+// NewCommitLogBootstrapper creates a new bootstrapper to bootstrap from commit log files.
+func NewCommitLogBootstrapper(
+	prefix string,
```

Comment:
Probably more consistent to get the prefix from the database options via `GetFilePathPrefix()` instead of explicitly pass it in?